src/components/productoverview/ProductList.vue

```diff
@@ -36,6 +37,18 @@ export default {
 
   computed: {
     category: vm => vm.categories.results[0],
+
+    isLoading() {
+      return this.$apollo.loading;
+    },
+
+    passSorting() {
+      if (this.sort === 'newest') {
+        return 'createdAt desc';
+      } if (this.sort === 'oldest') {
+        return 'createdAt asc';
+      } return null;
+    },
```

Comment:
This will eventually grow a lot. And `switch` is a better fit for the structure you are using.